src/Tizen.Applications.ComponentBased.Port/Tizen.Applications.ComponentBased.Port/ComponentTask.cs

```diff
@@ -0,0 +1,102 @@
+﻿/*
+ * Copyright (c) 2021 Samsung Electronics Co., Ltd All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+using System.Threading;
+
+namespace Tizen.Applications.ComponentBased
+{
+    /// <summary>
+    /// Provides a task class for the ComponentPort class.
```

Comment:
I'm not sure about the "task class", but I don't have a better suggestion. For me it looks like a convenience class to manage ComponentPort communication in a separate thread.

Reply:
Yes. It's for that. To use ComponentPort properly, it should be used in the separate thread.

Reply:
Then, please change the description to add this information.

Reply:
Done